dojo.py

```diff
@@ -107,7 +149,8 @@ def do_add_person(self, args):
                 print "Job can only be Staff or Fellow. Not case sensitive."    
         else:
             print "Please use both first and last names."    
-                
+       
```

Comment:
if checks that the user inputs second name. The second name cannot be the values of the role.